pkg/subgraph/src/mappings/cv-strategy.ts

```diff
@@ -246,6 +248,47 @@ export function handleSupportAdded(event: SupportAdded): void {
   cvp.save();
 }
 
+export function handlePointsDeactivated(event: PointsDeactivated): void {
+  let member = Member.load(event.params.member.toHexString());
+
+  if (member !== null) {
+    const stakes = member.stakes.load();
+    for (let i = 0; i < stakes.length; i++) {
+      const proposal = CVProposal.load(stakes[i].proposal);
+      if (proposal !== null) {
+        const strategy = CVStrategy.load(proposal.strategy);
+        if (strategy !== null) {
+          if (strategy.id == event.address.toHexString()) {
+            const stake = Stake.load(stakes[i].id);
+            if (stake !== null) {
+              const stakedAmount = stake.amount;
+              stake.amount = BigInt.fromI32(0);
+              stake.save();
+              proposal.stakedAmount = proposal.stakedAmount.minus(stakedAmount);
+              const cvc = CVStrategyContract.bind(event.address);
+              const contractProposal = cvc.getProposal(proposal.proposalNumber);
```

Comment:
search for `try_getProposal` in the code, i think that should use that because can revert, if that dont exist anymore in code you can keep it as is.